src/Adapters.Tests/Etw.Net451.Tests/Etw.Net451.Tests.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{15FA58D1-F397-4808-831C-7AE65EE640AF}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Microsoft.ApplicationInsights.EtwCollector.Tests</RootNamespace>
+    <AssemblyName>Microsoft.ApplicationInsights.EtwCollector.Net451.Tests</AssemblyName>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
Could you please elaborate the concerns that 4.6 is used in the Test project? I use 4.6 is because the mocked EventSource I use for the unit tests provides richer info (mainly, the payload names) that we can do more check over the event data.